screens/search_screen/search.tsx

```diff
@@ -24,23 +26,47 @@ const Search: FC<SearchScreenProps> = (props) => {
     (async () => {
       let { status } = await Location.requestForegroundPermissionsAsync();
       if (status !== "granted") {
-        alert("Permission to access location was denied");
+        alert("Permission to access location was denied"); useEffect(() => {
+          (async () => {
+            let { status } = await Location.requestForegroundPermissionsAsync();
+            if (status !== "granted") {
+              alert("Permission to access location was denied");
```

Comment:
I think we want to use the `Alert` component here instead.  Is there a reason we have to use the function?